tools/hrm_tools.py

```diff
@@ -53,11 +55,9 @@ def autocorr_freq(signal, fs):
         return
     ret = fftconvolve(signal, signal[::-1], mode='full')
     # get rid of negative lags
-    last_half = ret[ret.size//2:]
-    corr = last_half[last_half.size//2:]
-    d = np.diff(last_half)
+    corr = ret[ret.size//2:]
+    d = np.diff(corr)
     start = find(d > 0)[0]
     peak = np.argmax(corr[start:]) + start
-    # sampling frequency
     freq = fs/peak
-    return freq
+    return int(freq)
```

Comment:
Is there a reason why you expect `freq` to be an `int`?

Reply:
I wasn't sure how else to test it without rounding it down! I could round it in the testing function...?

Reply:
can you not assert something to a floating point value?

Reply:
It isn't always 3.0, it is usually some decimal value